src/components/visualizations/chart/highcharts/plugins/zeroAlignYAxis.ts

```diff
@@ -104,7 +104,7 @@ function getRange(targetNumber: number, delta: number): number[] {
     let position = 0;
     const result = [];
 
-    const limit = Math.abs(targetNumber);
+    const limit = +Math.abs(targetNumber).toFixed(9);
```

Comment:
we should define constant for `9`
and why do we need `+`?

Reply:
`toFixed` returns a string and plus sign will change the result to number